src/components/Authorization/Authorization.jsx

```diff
@@ -32,114 +33,168 @@ export const Authorization = ({primary}) => {
     event.preventDefault();
     setEmailError("");
     setPasswordError("");
-
+    let isValid = true;
     if (!email) {
       setEmailError("Это поле обязательно для заполнения");
-    } else if (!validateEmail(email)) {
-      setEmailError("Вы заполняете поле в неверном формате");
+      isValid = false;
     }
     if (!password) {
       setPasswordError("Это поле обязательно для заполнения");
-    } else if (!validatePassword(password)) {
-      setPasswordError("Пароль должен содержать минимум 8 символов");
+      isValid = false;
     }
-    if (emailError || passwordError) {
-      setShowSuccessMessage(false);
+    if (!isValid) {
       return;
     }
-    // @ts-ignore
-    dispatch(authentication({
-      userEmail: email,
-      userPassword: password
-    }))
-  }
-    const onEmailChange = (event) => {
+    try {
+      // @ts-ignore
+      await dispatch(
+        authentication({
+          userEmail: email,
+          userPassword: password,
+        }),
+      );
+    } catch (error) {
+      setEmailError("Неверный логин или пароль");
+      setPasswordError("Неверный логин или пароль");
```

Comment:
Why we use something like this? For what?

Reply:
fix